### PR TITLE
[Instrument List] Fixed questionable code so it stops spamming error logs

### PR DESCRIPTION
**Pull request by AnyhowStep**
Accessing `/instrument_list/?candID=XXXXXX&sessionID=XXXX` always generates three error log messages with warnings about undefined indices. This pull request stops them from appearing without modifying the behaviour of the code.

I plan to go through the code base and fix these little things where ever I see them.

### CODE REVIEW

File: php/libraries/BVL_Feedback_Panel.class.inc [hunk 1]
```diff
@@ -62,16 +62,21 @@ class BVL_Feedback_Panel
...
-            ->_feedbackObjectInfo["CommentID"];
+        $feedbackProfile = $this->_feedbackThread->_feedbackCandidateProfileInfo;
+        $candidateID     = $feedbackProfile["CandID"];
+
+        $this->tpl_data['candID'] = $candidateID;
+
+        $candidateObject         = Candidate::singleton($candidateID);
+        $this->tpl_data['pscid'] = $candidateObject->getPSCID();
+
+        $this->tpl_data['sessionID'] = isset($feedbackProfile["SessionID"]) ?
```
Comment (driusan) [inline php/libraries/BVL_Feedback_Panel.class.inc]: This should be `??` for readability, not `? .. :`

File: php/libraries/BVL_Feedback_Panel.class.inc [hunk 2]
```diff
@@ -62,16 +62,21 @@ class BVL_Feedback_Panel
...
+            $feedbackProfile["SessionID"] : null;
+
+        $feedbackObject = $this->_feedbackThread->_feedbackObjectInfo;
+        $this->tpl_data['commentID'] = isset($feedbackObject["CommentID"]) ?
```
Comment (driusan) [inline php/libraries/BVL_Feedback_Panel.class.inc]: This should be `??`, not `? .. :`

File: php/libraries/BVL_Feedback_Panel.class.inc (hunk 1 above)
Comment (AnyhowStep) [inline php/libraries/BVL_Feedback_Panel.class.inc]: This was written 28 days ago...
Will change, anyway.

Comment (AnyhowStep) [inline php/libraries/BVL_Feedback_Panel.class.inc]: Weird. I don't remember writing any of this, lol. But Github says it's my pull request. I guess it is...

**Pull request merged**